lorien/UI/Palette.gd

```diff
@@ -0,0 +1,66 @@
+extends Control
+class_name Palette
+
+signal color_changed
+
+# -------------------------------------------------------------------------------------------------
+export var _default_button_number: int
+export var _palette_theme: Theme
+export var _color_picker_path: NodePath 
+export(String, "Lorien-Default","Ice-Cream") var _default_palette: String 
+
+onready var _button_grid: GridContainer = $ScrollContainer/GridContainer
+onready var _color_picker: ColorPicker = get_node(_color_picker_path)
+onready var _palette_button_theme_normal: StyleBoxFlat = _palette_theme.get_stylebox("normal", "Button")
+onready var _palette_button_theme_focus: StyleBoxFlat = _palette_theme.get_stylebox("focus", "Button")
+
+var _palette_button: PackedScene = preload("res://UI/Components/PaletteButton.tscn")
```

Comment:
Make this a `const` and all upper case like: `PALETTE_BUTTON`. Also put it at the top of the file kind of like an import/include in other programming languages. See `InfiniteCanvas.gd` for reference.